solace-spring-boot-starters/solace-java-spring-boot-starter/README.md

```diff
@@ -70,43 +70,9 @@ final JCSMPSession session = solaceFactory.createSession();
 
 The `SpringJCSMPFactory` is a wrapper of the singleton `JCSMPFactory` which contains an associated `JCSMPProperties`. This facilitates auto-wiring by Spring but otherwise maintains the familiar `JCSMPFactory` interface known to users of the Solace Java API.
 
-Alternatively, you could autowire one or more of the following to create your own customized `SpringJCSMPFactory`:
-
-```java
-/* A factory for creating SpringJCSMPFactory. */
-@Autowired
-private SpringJCSMPFactoryCloudFactory springJcsmpFactoryCloudFactory;
-
-/* A POJO describing the credentials for the first detected Solace PubSub+ service */
-@Autowired
-private SolaceServiceCredentials solaceServiceCredentials;
-
-/* The properties of a JCSMP connection for the first detected Solace PubSub+ service */
-@Autowired
-private JCSMPProperties jcsmpProperties;
```

Comment:
Auto-wiring `JCSMPProperties` can remain in the `README.md` since its still a valid use case.